src/views/TabView/TabBarBottom.js

```diff
@@ -51,11 +53,15 @@ type Props = {
   tabStyle?: ViewStyleProp,
   showIcon?: boolean,
   isLandscape: boolean,
+  layout: Layout,
+  adaptiveTabs: boolean,
```

Comment:
I think just `adaptive` is fine. `Tabs` is redundant? Or change to `adaptiveLabels`